erpnext/buying/doctype/supplier_scorecard/supplier_scorecard.js

```diff
@@ -0,0 +1,161 @@
+// Copyright (c) 2017, Frappe Technologies Pvt. Ltd. and contributors
+// For license information, please see license.txt
+
+/* global frappe, refresh_field */
+
+frappe.ui.form.on("Supplier Scorecard", {
+
+	onload: function(frm) {
+		if (frm.doc.indicator_color !== "")
+		{
+			frm.set_indicator_formatter("status", function(doc) {
+				return doc.indicator_color.toLowerCase();
+			});
+		}
+		if (frm.doc.__unsaved == 1)
+		{
```

Comment:
style guide: We use this brace at the end of `if` condition in the same line. Please fix this everywhere.